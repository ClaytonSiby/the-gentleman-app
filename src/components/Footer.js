import React from 'react';

const Footer = () => (
  <div className="p-0">
    <h2>This is the footer!</h2>
  </div>
);

export default Footer;
