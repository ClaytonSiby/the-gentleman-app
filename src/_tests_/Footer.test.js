import React from 'react';
import { render } from '@testing-library/react';
import renderer from 'react-test-renderer';
import Footer from '../components/Footer';

describe('Footer', () => {
  test('Footer renders successfully', () => {
    render(<Footer />);
  });

  test('Renders consistently to the DOM', () => {
    const test = renderer.create(<Footer />).toJSON();

    expect(test).toMatchSnapshot();
  });
});
