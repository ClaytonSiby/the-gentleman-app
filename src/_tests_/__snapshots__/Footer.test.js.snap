// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Renders consistently to the DOM 1`] = `
<div
  className="mt-2 pt-4 text-center footerContainer"
>
  <p>
    Developed by Clayton Siby. Â©copyright 2021. All rights reserved.
  </p>
  <p>
    <span
      className="socialIcon"
    >
      <a
        href="https://github.com/ClaytonSiby"
        rel="noreferrer"
        target="_blank"
      >
        <i
          className="fab fa-github"
        />
      </a>
    </span>
    <span
      className="socialIcon"
    >
      <a
        href="https://www.linkedin/in/clayton-siby"
        rel="noreferrer"
        target="_blank"
      >
        <i
          className="fab fa-linkedin-in"
        />
      </a>
    </span>
    <span
      className="socialIcon"
    >
      <a
        href="https://twitter.com/ClaytonSiby"
        rel="noreferrer"
        target="_blank"
      >
        <i
          className="fab fa-twitter"
        />
      </a>
    </span>
  </p>
</div>
`;
